forch/forchestrator.py

```diff
@@ -72,7 +72,7 @@
 SEGMENTS_VLANS_FILE = 'segments_vlans_file'
 TAIL_ACL_CONFIG = 'tail_acl_config'
 SEQUESTER_SEGMENT_DEFAULT = 'SEQUESTER'
-SEQUESTER_TIMEOUT_DEFAULT = 20 * 60
+SEQUESTER_TIMEOUT_SEC_DEFAULT = 20 * 60
```

Comment:
Normally the _SEC suffix would go at the end since it's most closely related to the value (20*60) and the concept of SEQUESTER_TIMEOUT_DEFAULT could (theoretically) be mapped to different units (e.g. _MIN or _MS).  The default for SEC would never be different than the default for MIN.  SEQUESTER_SEGMENT_SEC and SEQUESTER_SEGMENT_DEFAULT_SEC could be different values so the affinity is less (although I understand you're applying the _DEFAULT suffix to the parameter SEQUESTER_TIMEOUT).

I can't find anything that explicitly says this is the right way... but a quick Google CS search shows 29k hits for _DEFAULT_SEC and only ~300 for _SEC_DEFAULT, so... at least that's the prevailing convention.

Reply:
I think it should be DEFAULT_SEQUESTER_TIMEOUT_SEC but that doesn't go along with SEQUESTER_SEGMENT_DEFAULT which was why I opted for SEQUESTER_TIMEOUT_SEC_DEFAULT to match. If you agree DEFAULT_SEQUESTER_TIMEOUT_SEC is better, I can change SEQUESTER_SEGMENT_DEFAULT to DEFAULT_SEQUESTER_SEGMENT as well.